### PR TITLE
Add docker information

### PR DESCRIPTION
**Pull request by lordtatty**
I've tested this with the "URL Tool" and "Add an Item".  This was asked about recently by a potential user of the tool.  However as we know it's likely only a small subset of users who will be able to use docker in their environments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
 # TARL Bulk Tools
 
+This guide will demonstrate two methods of running this tool locally - Docker and XAMPP.
+
+## Running with Docker
+
+If you already have [Docker](https://docs.docker.com/get-docker/) installed/running on your machine then run the following command from the root directory of this project to start this tool:
+
+```
+docker run -d -p 80:80 -v "$PWD":/var/www/html php:7.2-apache
```
Comment (timhodson) [inline README.md:10]: Might be worth an explanation that this mounts your local folder as a volume in the docker guest machine and so any changes locally will be reflected in the docker machine?

Also the user will need to dip back into the xampp instructions at a later step to complete the user config setup?

Comment (lordtatty) [reply README.md:10]: Good points.  I don't want to overcomplicate with too detailed an explanation of Docker, but I will add something to cover it...

**Comment by lordtatty** [comment]
Question - should this information be at the bottom of the README file?  I put it at the top as it's short, but maybe should be at the bottom as it's less likely to be used...

**Comment by lordtatty** [comment]
Oh, it got merged :D

**Pull request merged**